### PR TITLE
add feat: .from_csv, stat tracker and runner

### PR DESCRIPTION
**Pull request by stephenfabian**
Added .from_csv method, stat tracker file, stat tracker spec, and runner.  Collabed over zoom.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'csv'
+
+class StatTracker
+    attr_reader
```
Comment (jasonnoble) [inline lib/stat_tracker.rb]: Doesn't `attr_reader` take an argument?

**Review by aleish-m — APPROVED** [review]

File: lib/stat_tracker.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+ 
+    def initialize
```
Comment (jasonnoble) [inline lib/stat_tracker.rb]: If there's nothing to do here, we shouldn't need this method (use Ruby's default)

**Review by KevinT001 — APPROVED** [review]
Good Zoom session with the group to get project started, so we are all on the same page. Thanks for doing the first pull request! Pulling to my local main now

File: spec/stat_tracker_spec.rb [hunk 3]
```diff
@@ -0,0 +1,24 @@
+require './lib/stat_tracker.rb'
+require_relative 'spec_helper.rb'
+
+
+describe StatTracker do
+
+    describe '.from_CSV(locations)' do
```
Comment (jasonnoble) [inline spec/stat_tracker_spec.rb]: `.from_csv(locations)`

Comment (stephenfabian) [reply spec/stat_tracker_spec.rb]: made CSV lowercase

File: lib/stat_tracker.rb (hunk 2 above)
Comment (stephenfabian) [reply lib/stat_tracker.rb]: deleted

File: lib/stat_tracker.rb (hunk 1 above)
Comment (stephenfabian) [reply lib/stat_tracker.rb]: deleted attr_reader

**Pull request merged**